ssl_checker.py

```diff
@@ -0,0 +1,68 @@
+import socket
+import ssl
+from datetime import datetime, timedelta
+from typing import Dict, Union
```

Comment:
Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

Also it's a good practise to group imports im alphabet order per such a block